packages/material-ui-system/src/borders.js

```diff
@@ -44,10 +47,30 @@ export const borderColor = style({
   themeKey: 'palette',
 });
 
-export const borderRadius = style({
-  prop: 'borderRadius',
-  themeKey: 'shape',
-});
+function resolveCssProperty(props, prop, transformer) {
+  // Using a hash computation over an array iteration could be faster, but with only 28 items,
+  // it's doesn't worth the bundle size.
```

Comment:
```suggestion
  // it's isn't worth the bundle size.
```

Reply:
```suggestion
  // it isn’t worth the bundle size.
```

I think this is the correct wording :)

Reply:
Yep, missed that!